cocktails9/cocktails9/ContentView.swift

```diff
@@ -9,13 +9,9 @@ import SwiftUI
 
 struct ContentView: View {
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundStyle(.tint)
-            Text("Hello, world!")
-        }
-        .padding()
+        let cocktailService = CocktailService()
+        let filterService = FilterService()
+        CocktailsGridView(cocktailService: cocktailService, filterService: filterService)
```

Comment:
I imagine initialization of CocktailsGridView will be moved to main tab view after merge?